{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"provenance":[],"machine_shape":"hm","authorship_tag":"ABX9TyOqhRpBLGB6VR4d08GJLmGs"},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"},"accelerator":"TPU"},"cells":[{"cell_type":"markdown","metadata":{"id":"_kF4uur5CqEL"},"source":["## Mount Drive"]},{"cell_type":"code","execution_count":1,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":15463,"status":"ok","timestamp":1693473863436,"user":{"displayName":"Harrison Field","userId":"02417387657116967688"},"user_tz":-60},"id":"dM3x5UhDh7Bv","outputId":"169ad731-5d11-4595-c3be-0f27c04473f4"},"outputs":[{"output_type":"stream","name":"stdout","text":["Mounted at /content/drive\n"]}],"source":["from google.colab import drive\n","drive.mount('/content/drive')"]},{"cell_type":"code","execution_count":2,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":229,"status":"ok","timestamp":1693473975766,"user":{"displayName":"Harrison Field","userId":"02417387657116967688"},"user_tz":-60},"id":"sVvsrA9Hjl8z","outputId":"5c9353d3-21dc-456c-fd47-0783fdb10ed3"},"outputs":[{"output_type":"stream","name":"stdout","text":["/content/drive/MyDrive/baseline-reruns\n"]}],"source":["%cd /content/drive/MyDrive/baseline-reruns/"]},{"cell_type":"code","source":["!pip install keras sklearn"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"7faCsj2m1iQI","executionInfo":{"status":"ok","timestamp":1693305151951,"user_tz":-60,"elapsed":1217,"user":{"displayName":"Harrison Field","userId":"02417387657116967688"}},"outputId":"7b54c7f7-91d3-4aae-c017-9f06710be77e"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["Requirement already satisfied: keras in /usr/local/lib/python3.10/dist-packages (2.12.0)\n","Collecting sklearn\n","  Downloading sklearn-0.0.post7.tar.gz (3.6 kB)\n","  \u001b[1;31merror\u001b[0m: \u001b[1msubprocess-exited-with-error\u001b[0m\n","  \n","  \u001b[31m×\u001b[0m \u001b[32mpython setup.py egg_info\u001b[0m did not run successfully.\n","  \u001b[31m│\u001b[0m exit code: \u001b[1;36m1\u001b[0m\n","  \u001b[31m╰─>\u001b[0m See above for output.\n","  \n","  \u001b[1;35mnote\u001b[0m: This error originates from a subprocess, and is likely not a problem with pip.\n","  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25herror\n","\u001b[1;31merror\u001b[0m: \u001b[1mmetadata-generation-failed\u001b[0m\n","\n","\u001b[31m×\u001b[0m Encountered error while generating package metadata.\n","\u001b[31m╰─>\u001b[0m See above for output.\n","\n","\u001b[1;35mnote\u001b[0m: This is an issue with the package mentioned above, not pip.\n","\u001b[1;36mhint\u001b[0m: See above for details.\n"]}]},{"cell_type":"code","source":["!python NewTrainRes-Pouring.py"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"3VdUxCnt1vny","outputId":"dc76a4f4-4c98-460c-c1a8-6610cb00b703"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["2023-08-31 09:26:19.995855: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.\n","To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.\n","2023-08-31 09:26:21.121139: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT\n","Using real-time data augmentation.\n","Classify Phase!\n","Train Size: (4552, 4) Test Size: (1138, 4)\n","ResNetv1\n","file path for model: /content/drive/MyDrive/baseline-reruns/Log/Robot_ResNetv1_model.{epoch:03d}.h5\n","WARNING:tensorflow:`period` argument is deprecated. Please use `save_freq` to specify the frequency in number of batches seen.\n","WARNING:tensorflow:`period` argument is deprecated. Please use `save_freq` to specify the frequency in number of batches seen.\n","/content/drive/MyDrive/baseline-reruns/NewTrainRes-Pouring.py:379: UserWarning: `Model.fit_generator` is deprecated and will be removed in a future version. Please use `Model.fit`, which supports generators.\n","  model.fit_generator(datagen.flow(X_train, Y_train, batch_size=Config.batch_size),\n","2023-08-31 09:26:27.050955: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype int32\n","\t [[{{node Placeholder/_0}}]]\n","Epoch 1/200\n","227/227 [==============================] - 37s 127ms/step - loss: 1.1701 - accuracy: 0.8299 - val_loss: 0.8410 - val_accuracy: 0.8946 - lr: 0.0010\n","Epoch 2/200\n","227/227 [==============================] - 28s 123ms/step - loss: 0.7962 - accuracy: 0.8906 - val_loss: 0.6951 - val_accuracy: 0.8998 - lr: 0.0010\n","Epoch 3/200\n","227/227 [==============================] - 26s 116ms/step - loss: 0.6856 - accuracy: 0.8914 - val_loss: 0.6067 - val_accuracy: 0.8998 - lr: 0.0010\n","Epoch 4/200\n","100/227 [============>.................] - ETA: 14s - loss: 0.6204 - accuracy: 0.8905"]}]},{"cell_type":"code","source":["from google.colab import runtime\n","runtime.unassign()"],"metadata":{"id":"7GvrXnug2CLJ"},"execution_count":null,"outputs":[]},{"cell_type":"code","source":[],"metadata":{"id":"TmgRa1amSjF7"},"execution_count":null,"outputs":[]}]}